letsencrypt/display/util.py

```diff
@@ -403,6 +411,117 @@ def _get_valid_int_ans(self, max_):
 
         return OK, selection
 
+class NoninteractiveDisplay(object):
```

Comment:
It could, and reuse `notification` that way.  Or it could just call that `notification` function.  Would save about 4 lines of code, not sure it's worth the compexity.